.flake8

```
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 160
+max-complexity = 25
+ignore = E121,E722
```

コマンドラインオプションで指定されていたオプションは `.flake8` に移動しました。autopep8 も [`.flake8` を読んでくれるようです](https://github.com/hhatto/autopep8/tree/v1.6.0#configuration)。

---

`E722 do not use bare except` は厳しすぎるのと該当箇所が多いため、今回新たに ignore しています。